### ISSUE TITLE
Creating a new subtool instead of updating original

### ISSUE THREAD

**Issue by barbadul** [JoseConseco/GoB#53]
Hello! 
This addon is very good, but I've got an annoying problem. After I press GoZ in ZBrush, my mesh imports to my scene in Blender. Then, even with no changes at all, after I click on Export in Blender -- it creates a new subtool in Zbrush with the same name. I've tried lots of things -- cleaning cache, creating new scenes and so on but nothing helps. I've tested with both 2018 and 2019.1 versions of Zbrush. I've got latest Steam version of 2.8 blender and latest addon version. Even renamed archive and folder inside to "GoB", nothing changes.

And the most interesting: after I've got a duplicate of my mesh in Zbrush, I can GoZ this duplicate to and from Blender with no problems at all -- everything works well. So the workaround I am using right now is to GoZ mesh twice to Blender and back and only after this I can start tweaking mesh in Blender with no risks ZBrush will not recognize it. I really hope this is possible to fix cause I keep forgetting about this workaround and suffer a lot :^)

Thanks for your amazing work, I really appreciate it!

**Comment by schroef** [comment]
Perhaps screenshots of mesh objects or model objects could help here.

**Comment by barbadul** [comment]
> Perhaps screenshots of mesh objects or model objects could help here.

It is the same with any mesh or primitive, doesn't matter. But if you need any more details, I'll send them, sure.

**Comment by schroef** [comment]
Well im not the dev, but a lot of times these kind of things can help and possible show the errors

**Comment by barbadul** [comment]
> Well im not the dev, but a lot of times these kind of things can help and possible show the errors

Sure, I just simply do not know which type of information should I post. As I said, it happened to me with any mesh, so the screens are pretty useless here. I run Windows 10, Zbrush 2019.1 and latest 2.8 Steam version + latest GoB Addon. AMD Ryzen 2600, GTX970, 16GB of RAM

**Comment by kromar** [comment]
@barbadul  did you enable the option in zbrush to import as subtool? 
![image](https://user-images.githubusercontent.com/1472884/68077231-7598b100-fdc0-11e9-9bbc-bf574d27359b.png)

i remember i had this happen to me at some point but i was able to resolve it, i think my usual routine is to use the GoZ repair executable and then reconfigure zbrush and then restat zbrush and usually that helps.
![Screen Shot 11-02-19 at 10 02 PM](https://user-images.githubusercontent.com/1472884/68077248-c6100e80-fdc0-11e9-9b95-eea724f738d9.PNG)

**Comment by barbadul** [comment]
> @barbadul did you enable the option in zbrush to import as subtool?
Yes, I did 
> i remember i had this happen to me at some point but i was able to resolve it, i think my usual routine is to use the GoZ repair executable and then reconfigure zbrush and then restat zbrush and usually that helps.
I'll try to use GoZ repair right now, thanks

**Comment by barbadul** [comment]
Well,　I've tried to reinstall GoZ for ZBrush as you suggested and now it is got broken even more. GoZ to Blender works fine, but Export button of GoB doesn't work at all :(

**Comment by barbadul** [comment]
Looks like it depends on the version of this GoZ installer. I've reinstalled GoZ from 2018 ZBrush and the situation went back as it was. It is still annoying, but at least I can work with it

**Comment by barbadul** [comment]
https://youtu.be/5vshwa6gyJg Here I've attached a video of the problem if its gonna help somehow

**Comment by barbadul** [comment]
Looks like it started to work better now. I switched versions of both GoZ and GoB from time to time but the bug was still there. Right now I've updated everything back to the latest releases and haven't faced the bug yet for one day. Looks like the problem is pretty rare and random, not many people faced it. I am not sure about the solution, but after trying different configurations of GoZ and GoB it  kinda fixed itself...I guess. I am closing the thread then. Thanks to everybody for ideas!
